### ISSUE TITLE
Just a question of size

### ISSUE THREAD

**Issue by lpgpd7m** [astromatic/visiomatic#9]
Hello Mr Bertin,
Is it possible to deal with layers of the same object but with different sizes? In other words, is it possible to switch from a full size image to a smaller one on a same map? The smaller one would for example be a processed picture of an object of interest..
Thank you for your response,
David

**Comment by ebertin** [comment]
Yes you can switch from one layer to another, even if they don't share the same projection or pixel grid size. During the switch the world coordinates at the center of the screen are preserved.
_However_ VisiOmatic V2 still lacks a proper dialog to manage that switch (contributions welcome!).
You may also synchronize several maps together as in [this V1 example](http://image.iap.fr/demo/multi.html). I haven't tested it but it should still work with V2.
Best wishes

**Comment by lpgpd7m** [comment]
Thanks for your prompt answer. I did the test with the extra maps and they are indeed well synchronized although they have a different pixel grid size. Then, I guess it should not be so difficult to have the same possibility on the main map.. Do you confirm that one needs to change the visiomatic source code to allow this option (and not only change one of the options of the L.tileLayer.iip in the php file)? If so, could you please precise us which javascript should be modified?  Thanks in advance for your help,
David
